internal/pkg/template/templates/workloads/partials/cf/publish.yml

```diff
@@ -2,19 +2,21 @@
 {{- range $topic := .Publish.Topics}}
 {{logicalIDSafe $topic.Name}}SNSTopic:
   Metadata:
-    'aws:copilot:description':  {{- if eq $topic.Type "fifo" }} 'A FIFO SNS topic to broadcast {{$topic.Name}} events' {{- else}} 'A SNS topic to broadcast {{$topic.Name}} events'{{- end}}
+    'aws:copilot:description': {{ if or $topic.Fifo.Enable $topic.Fifo.ContentBasedDeduplication }} 'A FIFO SNS topic to broadcast {{$topic.Name}} events' {{ else}} 'A SNS topic to broadcast {{$topic.Name}} events'{{ end}}
```

Comment:
similar here, I think a method would simplify this logc that's in the template `$topic.IsFIFO()`